opendbc/car/interfaces.py

```diff
@@ -375,28 +375,43 @@ def update(self, CC: structs.CarControl, CS: CarStateBase, now_nanos: int) -> tu
 
 # interface-specific helpers
 
+@cache
+def _get_brand_names() -> list[str]:
+  return [folder.split('/')[-1] for folder in sorted(x[0] for x in os.walk(BASEDIR))]
+
+
+@cache
+def _get_brand_values_for_attr(brand_name: str, attr: str) -> Any | None:
+  attr_file = INTERFACE_ATTR_FILE.get(attr, "values")
+  try:
+    return __import__(f'opendbc.car.{brand_name}.{attr_file}', fromlist=[attr])
+  except (ImportError, OSError):
+    pass
+  return None
```

Comment:
You could import each values file once instead of several per brand, further reducing the import calls.

Reply:
As in, look up the `attr_file` and then cache based on that rather than the raw `attr`? Because if the `attr_file` is different, then I assume the imports aren't overlapping, right?